.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+(Title field)
+[OFFAPPS-XXX] Short description
+
+## Description
+Develop what, but also why this PR exists. The "Why" is as much, if not more important than the "What". I use this approach very often when I write commit messages, because if the code is clear enough it should already answer the "what" question. The "why" is often more subtle.
+
+## Tasks
+- [ ] Write tests
+
+## Implementation notes
+I don't always use this section, but I do when I find there's something particularly tricky end/or nifty in the code.
+
+## References
+[JIRA ticket](https://zendesk.atlassian.net/browse/OFFAPPS-XXX)
+[Invision design](https://google.com.au)
+
+## Screenshots (if needed)
+### Screenshot title
+However many images needed
```

Comment:
@hoangtrvu shared this nifty tip in Keith's PR on the user data app. I like it, I think it's a worthy addition, as it makes the images collapsible. Useful when many screenshots are needed.
```
## Screenshots (if needed)
<details>
<summary>Screen shot 1</summary>
<img src="1.png">
</details>

<details>
<summary>Screen shot 2</summary>
<img src="2.png">
</details>
```

Reply:
👍